Modules/IO/GDCM/include/itkGDCMImageIO.h

```diff
@@ -384,7 +403,7 @@ class ITKIOGDCM_EXPORT GDCMImageIO : public ImageIOBase
   /** defines whether this image is a 2D out of a 2D image
    *  or a 2D out of a 3D image. */
   unsigned int     m_GlobalNumberOfDimensions;
-  TCompressionType m_CompressionType;
+  TCompressionType m_GDCMCompressionType;
```

Comment:
I purposely renamed here, because behavior changed, not to confuse with super-class compression. Super-class compression, eg Jpeg, can not work and we are using GDCM change transfer syntax filter to compress,